Documents/DevDoc.md

```diff
@@ -1 +1,61 @@
-# DevDoc
+# Welcome to OAPS-Jifry
+
+**OAPS-Jifry** is a re-implementation of [OAPS](https://github.com/youmowang/zjnuservice) by Yun Dabang, and Ni Zhongjie, as OOAD Fall 2018 course project, using [JSP](https://en.wikipedia.org/wiki/JavaServer_Pages) technology.
+
+
+# How to contribute
+
+Currently, this project is maintained by team **MIJC** (Mahdi,IBRAHIM,Jifry, and Clive) and everyone else is welcomed to join us and help us to improve the project by any means.
+
+## Add Your Information to the Contributor List
+
+Clone the main repository:
+
+    $git clone https://github.com/lanlab-org/OAPS-Jifry
+Edit the **README.md** file, append your:
+
+> github account - Name - Student ID
+
+to the contributor list.
+
+Then, add your changes to the master repository, 
+
+    $git add .
+commit the changes, 
+
+    $git commit -m "your commit message"
+   and push them to the master repository.
+
+    $git push origin master
```

Comment:
@MarshmallowClive 

I think you don't want people to change the central master branch directly.

If you let contributors to do that, you do not understand [the feature branching workflow](https://github.com/spm2020spring/TeamCollaborationTutorial/blob/master/team.rst) very well.

**Hui**


Hui

Reply:
We did mention that in "Follow the feature-branching workflow" section.

What we meant in this part that he/she can directly append their names on the list of contributors.

Reply:
> We did mention that in "Follow the feature-branching workflow" section.
> 
> What we meant in this part that he/she can directly append their names on the list of contributors.

@hema-001 
@TanakaMichelle 

Thanks. Now I understand.   

Hui